openapi/azure.sqldb/Package.md

```diff
@@ -0,0 +1,18 @@
+Connects to [Azure SQL DB API 2017-03-01-preview](https://docs.microsoft.com/en-us/azure/azure-sql/database/sql-database-paas-overview)from Ballerina
```

Comment:
Please add a space before the word from.

Reply:
Did the suggested changes.